### DISCUSSION TITLE
UseCase: Daemon for Background Processing without Client/Server

### DISCUSSION THREAD

**Discussion by fakharak** [swoole/swoole-src#4214] [Q&A] [answered]
With Swoole, i have been looking into it from Network (Services) perpective only, so far. However, i see that there are large number of Use Cases related to concurrent processing in background which do not require Network.

Currently, I want to write a background process as "daemon" which needs to process multiple images stored on a file-path, and i want to perform processing concnurrently. 

I know we we can execute Co\Run()  (Swoole Coroutines) independent of Server Context, but i also want to know quickly if we can emit Events with Swoole Time Ticker out of Server Context? and how to make the Swoole Process a "Daemon Process" ?

**Comment by sy-records** [comment, marked as answer]
see https://www.swoole.co.uk/docs/modules/swoole-process-daemon

**Reply by fakharak** [reply]
That is exactly what i was looking for. Thank you.

**Reply by fakharak** [reply]
For timer, we can use:
https://www.swoole.co.uk/docs/modules/swoole-timer
